CODEOWNERS

```
@@ -62,13 +62,13 @@
 /src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ClickOnce.targets @sujitnayak
 
 # Area-Watch
-/src/Assets/TestProjects/Watch*/     @captainsafia, @pranavkm, @mkArtakMSFT
-/src/Tests/dotnet-watch.Tests/       @captainsafia, @pranavkm, @mkArtakMSFT
-/src/Tests/Microsoft.AspNetCore.Watch.BrowserRefresh.Tests/       @captainsafia, @pranavkm, @mkArtakMSFT
-/src/BuiltInTools/   @captainsafia, @pranavkm, @mkArtakMSFT
+/src/Assets/TestProjects/Watch*/     @captainsafia @mkArtakMSFT
+/src/Tests/dotnet-watch.Tests/       @captainsafia @mkArtakMSFT
+/src/Tests/Microsoft.AspNetCore.Watch.BrowserRefresh.Tests/       @captainsafia @mkArtakMSFT
+/src/BuiltInTools/   @captainsafia @mkArtakMSFT
 
 # Compatibility tools owned by runtime team
 # Area-Compatibility
-/src/Compatibility/ @Anipik, @safern, @ericstj
-/src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/ @Anipik, @safern, @ericstj
-/src/Tests/Microsoft.DotNet.PackageValidation.Tests/ @Anipik, @safern, @ericstj
+/src/Compatibility/ @Anipik @safern @ericstj
+/src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/ @Anipik @safern @ericstj
+/src/Tests/Microsoft.DotNet.PackageValidation.Tests/ @Anipik @safern @ericstj
```

```suggestion
/src/Compatibility/ @ericstj @dotnet/area-infrastructure-libraries @joperezr 
/src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/ @ericstj @dotnet/area-infrastructure-libraries @joperezr 
/src/Tests/Microsoft.DotNet.PackageValidation.Tests/ @ericstj  @dotnet/area-infrastructure-libraries @joperezr 
```
Can we use this?  cc @joperezr @jeffhandley

---

Works for me. Alternatively, if @joperezr wants to be considered a "Consultant" for `area-infrastructure-libraries` and would be OK with receiving the notifications, we could add him to the team and use the team membership as the single source of truth. But that has the tradeoff of also getting notifications for issues/prs labeled with the area, which might not be desired.

---

This is for our package validation tasks in dotnet/sdk only, whereas area-infrastructure-libraries covers a broader set of things the team owns.

---

Thanks, committed this in to see if it resolves the errors. Feel free to add further suggestions if necessary.

---

Looks like `dotnet/area-infrastructure-libraries` is missing access perms.

---

> Alternatively, if @joperezr wants to be considered a "Consultant" for area-infrastructure-libraries and would be OK with receiving the notifications,

I'm fine with whatever you think is the better approach, I don't mind getting the extra notifications and filtering down only to the ones where my input is relevant.

---

Is there a way to retrigger that check?  I just changed the parent team of https://github.com/orgs/dotnet/teams/area-infrastructure-libraries to be npt which has write access to dotnet/sdk.  I think this should satisfy it.

---

There are a bunch of failing checks (essentially every one involving a team). I'm not sure if this is actually indicative of missing perms or if the check simply doesn't take into account transitive access. I think the check is a fairly new feature, so it may be the latter.

---

It's possible my change hasn't yet taken effect.  It's still "pending" in the github UI now that I check it.  IMO it's still better to use a team here so we don't run into cases where it gets stale like it did before.

---

I got that team re-parenting approved and it seems to have fixed it.